### DISCUSSION TITLE
Account Unlinked with primary email

### DISCUSSION THREAD

**Discussion by ashishjaiswal501** [community/community#144657] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

anyone can you help me that recntly i have unlinked my primary account from github and there are no other email . i have only username how can do resolve it and access the github account

**Comment by jessie-ro** [comment, marked as answer]
If you've unlinked your primary email and don’t have another email associated with your GitHub account, you’ll need to contact [GitHub Support](https://support.github.com/) directly. Provide them with as much information as possible, such as your username and any details that can verify your ownership of the account (e.g., past activity, repositories, or payment receipts, if applicable).

Since email is required for account recovery, they may guide you through steps to restore access or link a new email. I hope this helps, and you get your account access back soon!

**Comment by queenofcorgis** [comment]
Hi @ashishjaiswal501, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact) like @jessie-ro mentioned.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
